docs/specs/forecast-api.md

```diff
@@ -122,8 +120,8 @@ generated using the nearest forecast time.
 
 ### flight level
 
-The fl value represents the flight level (*hectofeet*) of the returned
-netCDF global grid.
+The {fl} value represents the flight level (*hectofeet*) of the returned
```

Comment:
Shouldn't the {fl} be in a code block, i.e. `{fl}`?

Reply:
yes; inline. I'll also scan for other similar fmt fixes